webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue

```diff
@@ -1,40 +1,126 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2022 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
 <template>
   <div class="d-flex flex-column white border-radius">
-    <img
-      :height="height"
-      :style="{maxHeight: height}"
-      src="/social-portlet/images/emptySpaceStream.png"
-      class="mx-auto"
-      loading="lazy">
-    <div class="mx-4 mb-4">
+    <div class="mx-4 my-4">
       <p v-sanitized-html="welcomeTitle"></p>
+      <v-card
+        max-width="500"
+        flat
+        class="mx-auto my-8">
+        <v-row>
+          <activity-stream-empty-message-space-icons-column
```

Comment:
How to choose which message to display ? (I mean there is not  `v-if-else` block, do you display four blocks at the same time ?)

Reply:
yes I need All of them to be displayed in this way https://community.exoplatform.com/portal/rest/jcr/repository/collaboration/Groups/spaces/meeds_labers/Documents/product/ux/placeholder%20stream%20page.png?time=1666885886654

Reply:
ok, thanks.